include/rocksdb/cleanable.h

```diff
@@ -88,7 +88,7 @@ class Cleanable {
 // It will prevent cleanups from ever happening!
 class SharedCleanablePtr {
  public:
-  // Empy/null pointer
+  // Empty/null pointer
```

Comment:
I think this comment can be removed because it does not provide any extra information.

Reply:
I feel the same way